read-fonts/src/lib.rs

```diff
@@ -257,4 +257,19 @@ impl<'a> TableProvider<'a> for FontRef<'a> {
     fn data_for_tag(&self, tag: Tag) -> Option<FontData<'a>> {
         self.table_data(tag)
     }
+
+    fn outline_format(&self) -> Option<OutlineFormat> {
+        Some(match self.table_directory.sfnt_version() {
+            TT_SFNT_VERSION => OutlineFormat::TrueType,
+            CFF_SFNT_VERSION => OutlineFormat::PostScript,
+            _ => return None,
```

Comment:
hmmm.

For FontRef, at least, if the table directory has an unknown sfnt version we return an error.

Are there other `TableProviders` for which this isn't true?

It might make sense to be maximally defensive, but I think it's probably reasonable to say we don't handle unknown `sfnt` values, since it's really not clear what we should do in that case?

Reply:
Good point. I could imagine a `TableProvider` impl that doesn't provide any outlines at all I suppose. The `table_directory` field on `FontRef` is also public so it's conceivable to construct one with an sfnt version that doesn't match what we expect.

One possible solution would be returning `PostScript` for `CFF_SFNT_VERSION` here and just defaulting to `TrueType` for any other values.